wled00/network.cpp

```diff
@@ -123,6 +123,16 @@ const ethernet_settings ethernetBoards[] = {
     18,                   // eth_mdio,
     ETH_PHY_LAN8720,      // eth_type,
     ETH_CLOCK_GPIO17_OUT  // eth_clk_mode
+  },
+
+  // ESP32-POE-WROVER
+  {
+    0,                    // eth_address,
+    12,                   // eth_power,
+    23,                   // eth_mdc,
+    18,                   // eth_mdio,
+    ETH_PHY_LAN8720,      // eth_type,
+    ETH_CLOCK_GPIO0_OUT   // eth_clk_mode
```

Comment:
How does this work when two pins are assigned to same GPIO (0)?

Reply:
I am quite new to the WLED source code. Is there any way to avoid that a GPIO used by ethernet is selectable for other components?

Reply:
Sorry, my bad, I wrongly interpreted `eth_address`. All is good.

Reply:
Another question: Do I have to commit the generated html_*.h files too?

Reply:
yes, but please re-base to 0_15 branch first.

Reply:
Done.